_scripts/cirrus.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -ex
+
+echo -e "Starting cirrus experimenter."
+
+docker run -it --name cirrus\
+  --net fxa \
+  --mount type=bind,source=$(pwd)/_scripts/configs/cirrus.env,target=/cirrus/.env \
```

Comment:
The source path doesn't exist.

Reply:
Nice catch. I realized now that we have a gitignore to avoid committing .env in case they have access tokens there. I'll rename this sample file instead to `cirrus.env.sample`.